docs/content/anchoredPosition.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Anchored Position Behavior
+---
+
+The `getAnchoredPosition` behavior and `useAnchoredPosition` hook are used to calculate the position of a "floating" element that is anchored to another DOM element. This is useful for implementing overlay UI, such as dialogs, popovers, tooltips, toasts, and dropdown-style menus.
+
+At a high level, the `getAnchoredPosition` routine will attempt to find the most suitable position for the floating element based on the passed-in settings, its containing element, and the size and position of the anchor element. Specifically, the calculated position should try to ensure that the floating element, when positioned at the calculated coordinates, does not overflow or underflow the container's bounding box.
+
+Settings for this routine allow the user to customize several aspects of this calculation. See **PositionSettings** below for a detailed description of these settings.
+
+### Floating element container
+
+The anchored position calculation is based on the floating element's closest _positioned_ ancestor. In other words, we try to check parents of the floating element until we find one that has a position set to anything other than `static` and use that element's bounding box as the container. If we can't find such an element, we will try to use `document.body`. This element should be large—big enough to accommodate the floating element and have plenty of space to be moved around. It may be a good idea to ensure that this container element _also_ contains the anchor element and is scrollable. This will ensure that when scrolled, the anchor and floating element will move together.
+
+### Demo
+
+See [this CodePen](https://codepen.io/team/GitHub/pen/OJbPKNZ) for a demo of `useAnchoredPosition`.
+
+### Usage
+
+```ts
+const settings = {
+  side: 'outside-right',
+  align: 'center',
+  alignmentOffset: 10,
+  anchorOffset: -10
+} as Partial<PositionSettings>
+const float = document.getElementById('floatingElement')
+const anchor = document.getElementById('anchorElement')
+const {top, left} = getAnchoredPosition(float, anchor, settings)
+float.style.top = `${top}px`
+float.style.left = `${left}px`
+```
+### PositionSettings
```

Comment:
There should also be a section for the rest of the parameters you can pass to `getAnchoredPosition`